src/prommis/solvent_extraction/sx_model_flowsheet.py

```diff
@@ -7,20 +7,32 @@
 )
 from idaes.core.util.model_statistics import degrees_of_freedom as dof
 
+from idaes.core.util import DiagnosticsToolbox
+from pyomo.util.check_units import assert_units_consistent
+
 from prommis.leaching.leach_solution_properties import LeachSolutionParameters
 from prommis.solvent_extraction.ree_og_distribution import REESolExOgParameters
+from prommis.solvent_extraction.ree_aq_distribution import REESolExAqParameters
 from prommis.solvent_extraction.solvent_extraction import SolventExtraction
 
+import matplotlib.pyplot as plt
+
 m = ConcreteModel()
-m.fs = FlowsheetBlock(dynamic=False)
+
+time_duration = 60
```

Comment:
Could you create separate dynamic and steady-state tests please. We need to have both to ensure the model runs correctly in both cases, and dynamic tests tend to be computationally expensive so should be run sparingly (thus we want the steady-state version to run more frequently).

Reply:
Fine I will create two separate tests in the model